local-test-infra/local-pod-under-test.yaml

```diff
@@ -5,8 +5,10 @@ metadata:
   labels:
     app: test
     test-network-function.com/generic: target
+    test-network-function.com/container: target
   annotations:
-    test-network-function.com/defaultnetworkinterface: "\"eth0\""
+    test-network-function.com/container_tests: '["PRIVILEGED_POD","PRIVILEGED_ROLE"]' # optional
+    test-network-function.com/defaultnetworkinterface: '"eth0"' # optional
```

Comment:
One more thing I've just noticed. Do we have a naming convention for TNF annotations? 

Some annotations use _snake case_ (e.g. `container_tests` or `skip_connectivity_tests`), while other annotations' names use no separator (`defaultnetworkinterface` instead of `default_network_interface`). 

Is this inconsistency intended?

Reply:
Another good point, not intended. However since this requires changes in two repositories, and would depend on some merged and some unmerged changes I would like to defer correcting this to reduce the possibility of mixups (though this increases the possibility of the change affecting developer clusters).